diff.py

```diff
@@ -1,24 +1,26 @@
 # -*- coding: utf-8 -*-
 
 
-def diff_line(i, j):
+def diff_line(i, j, ignore_hits):
     if j is None:
         return None
     elif i is None:
         return j
-    elif j > 0 and j >= i:
+    elif j > 0 and j >= i and ignore_hits is False:
         return j - i
```

Comment:
You can have a branch here instead:
```
elif j > 0 and j > i:
    if ignore_hits:
        return 1
    else:
        return j - i
else:
    return 0
```

Reply:
Seems that with such condition there won't be 0 if in baseline coverage is 1 and in the report is 5, for example.